app/assets/stylesheets/_variables.scss

```diff
@@ -0,0 +1,10 @@
+// Colors
+:root {
+  --color-original-black: #000;
+  --color-white: #fff;
+  --color-black: #15151a;
+  --color-white-180: rgba(255, 255, 255, 0.18);
+  --color-white-200: rgba(255, 255, 255, 0.2);
+  --color-white-600: rgba(255, 255, 255, 0.6);
+  --color-input-webkit-box-shadow: rgb(93, 91, 89);
+}
```

Comment:
I recommend keeping variables to define SASS variables as we currently do. Then create another file `_root.scss` in `base/`. This is similar to what is done in [Nimble website](https://github.com/nimblehq/nimblehq-web/blob/development/_sass/base/_root.scss) and Compass (same). The rationale is that variables can also be used in components and not necessarily need to be defined at the root element (for scoping reasons). So `:root` is for all "global" variables hence the base while variables can be used in all places and can be scoped.

Reply:
Thank you, I moved them to `base/_root`